src/IoTHub.Portal.Infrastructure/Services/LoRaWanManagementService.cs

```diff
@@ -40,7 +45,7 @@ public async Task<HttpResponseMessage> ExecuteLoRaDeviceMessage(string deviceId,
 
             var body = new LoRaCloudToDeviceMessage
             {
-                RawPayload = Convert.ToBase64String(Encoding.UTF8.GetBytes(commandDto.Frame)),
+                RawPayload = Convert.ToBase64String(Enumerable.Range(0, commandDto.Frame.Length / 2).Select(x => Convert.ToByte(commandDto.Frame.Substring(x * 2, 2), 16)).ToArray()),
```

Comment:
What is the objective here?

Reply:
Command were converted from string to Base64 with string as an ASCII string. But command are stored as Hex string.

0E10 was sent as 30453130

Reply:
So why not converting the commands in ASCII string before saving the command instead?

Reply:
@lucas-mrq let's discuss by teams to clarify that point, I don't really understand that point.